src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md

```diff
@@ -0,0 +1,423 @@
+for offline use when they have connectivity.
+
+Users on data plans may avoid downloading large files for fear of cost, so you may also want to
+display an associated cost so users can make an active comparison for a specific file or task. For
+example, if the music app above could detect if the user is on a data plan and show the file size so
+that users can see the cost of a file.
+
+### Help prevent hacked experiences
+
+Often users hack an experience without realizing they are doing it. For example before cloud sharing
+apps like Google Drive, it was common for users to save large files and attach them to emails so
+they could carry on editing from a different device. It is important not to be pulled into their
+hacked experience but rather look at what they are trying to achieve. In other words instead of
+thinking of how you can make attaching a large file more user friendly, solve the problem of
+sharing large files across multiple devices.
+
+## Make experiences transferable from one device to another
+
+When building for flaky networks, try to sync as soon as the connection improves
+so that the experience is transferable. For example, imagine a travel app losing
+a network connection mid-way through a booking. When the connection is reestablished, the app syncs
+with the user's account allowing them to continue their booking on their desktop device. Not being
+able to transfer experiences can be incredibly jarring to users.
+
+Inform the user of the current state of their data. For example, you could show whether the
+app has synced. Educate them where possible but try not to overburden them with messaging.
+
+## Create inclusive design experiences
+
+When designing seek to be inclusive by providing meaningful design devices, simple language,
+standard iconography, and meaningful imagery that will guide the user to complete the action or task
+rather than hinder their progress.
+
+### Use simple concise language
+
+Good UX is not just about a well designed interface. It includes the path a user takes as well as
+the words used in the app. Avoid tech jargon when explaining the state of the app or
+individual UI components. Consider that the phrase "app offline" might not convey to the user the
+current state of the app.
+
+<figure style="display: inline-block; max-width: 45%;" class="w-figure">
+  <img src="download.png" alt="A download icon is a good example">
+  <figcaption class="w-figcaption">
+    <b>DO</b>: Use language and imagery that describes the action.
+  </figcaption>
+</figure>
+
+<figure style="display: inline-block; max-width: 45%;" class="w-figure">
+  <img src="service-worker-ready.png" alt="A service worker icon is a bad example">
+  <figcaption class="w-figcaption">
+    <b>DON'T</b>: Avoid terms that aren't intelligible to non-technical users.
+  </figcaption>
+</figure>
+
+### Use multiple design devices to create accessible user experiences
```

Comment:
This section reinforces my previous suggestion to rephrase that section heading that is using the word `accessible` in a more general way

Reply:
In this context I think "accessible" is correct, since the text talks about color deficiencies.